aa_stripe/models.py

```diff
@@ -125,6 +125,47 @@ def create_at_stripe(self):
         self.save()
         return self
 
+    def update_at_stripe(self, stripe_token, should_be_default):
+        is_default = self.customer.default_card.pk is self.pk
```

Comment:
setting the card as default is not necessary on stripe (however it will not hurt), but it is necessary to update customer.default_card to the one if set_default=True

Reply:
I was unable to set default_source to "" or None on Stripes customer object and save it, 
and when updating a card via stripe.js it has to be a default card